rotkehlchen/data_import/importers/nexo.py

```diff
@@ -157,6 +171,35 @@ def _consume_nexo(
                 notes=f'{entry_type} from Nexo',
             )
             self.add_history_events(write_cursor, [event])
+        elif entry_type == 'Exchange':
+            input_asset = asset_from_nexo(csv_row['Input Currency'])
```

Comment:
All of these seems to also be able to raise KeyError all over the function right? The docstring does not mention it may raise it. Should it not mention it? Is it handled anywhere?

Reply:
key error is handled at here.
https://github.com/rotki/rotki/blob/5d9704524cdf87965c988868ea64e7632a0b5ead/rotkehlchen/data_import/importers/nexo.py#L168-L207